packages/calling-stateful-client/src/CallClientState.ts

```diff
@@ -1001,7 +998,6 @@ export interface CallClientState {
    * See documentation of {@Link CallErrors} for details.
    */
   latestErrors: CallErrors;
-  /* @conditional-compile-remove(breakout-rooms) */ /* @conditional-compile-remove(media-access) */
```

Comment:
Should we remove the  breakout room cc here?

Reply:
Yes, it's fine because latestNotifications is needed for hard mute